picard/coverart/providers/local.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+# Copyright (C) 2007 Oliver Charles
+# Copyright (C) 2007-2011 Philipp Wolfer
+# Copyright (C) 2007, 2010, 2011 Lukáš Lalinský
+# Copyright (C) 2011 Michael Wiencek
+# Copyright (C) 2011-2012 Wieland Hoffmann
+# Copyright (C) 2013-2014 Laurent Monin
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+import os
+import re
+import traceback
+
+from picard import config, log
+from picard.coverart.providers import CoverArtProvider
+from picard.coverart.image import CoverArtImage
+
+
+class CoverArtProviderLocal(CoverArtProvider):
+
+    """Get cover art from local files"""
+
+    NAME = "Local"
+
+    _match_re = re.compile('^(?:cover|folder).*\.(?:jpe?g|png|gif|tiff?)$', re.IGNORECASE)
```

Comment:
"AlbumArt" is also a typical name understood by many software. It's what I am using and it is pretty compatible. I remember switching to this naming for some reason, but I don't remember which software triggered it